west.yml

```diff
@@ -102,7 +102,7 @@ manifest:
       revision: 41132e9220f8bc1223084975350c5e5f3b492afe
       path: tools/net-tools
     - name: hal_nxp
-      revision: 61a7788ac9379bfa7711bf9d15de735230528ea5
+      revision: pull/63/head
```

Comment:
@MaureenHelm Is this correct one revision?

Reply:
The hal PR has now been merged, so this can be updated to the new SHA